src/test/java/at/ac/tuwien/kr/alpha/config/CommandLineParserTest.java

```diff
@@ -51,22 +51,22 @@
 	public void help() throws ParseException {
 		StringBuilder bld = new StringBuilder();
 		CommandLineParser parser = new CommandLineParser(DEFAULT_COMMAND_LINE, (msg) -> bld.append(msg));
-		parser.parseCommandLine(new String[] {"-h"});
+		parser.parseCommandLine(new String[] {"-h" });
```

Comment:
This is strange behaviour of your auto-formatter: why put the whitespace behind the string but not in front of it? Please disable this and revert to the previous state on all locations throughout this file, which is more consistent.

Reply:
Actually this is a weird quirk of our checkstyle config: My formatter formats as `parser.parseCommandLine(new String[] { "-h" });` i.e. with whitespace both at start and end, however, checkstyle complains about the whitespace after `{`.
Personally, I disagree with checkstyle and would be in favor of following my formatter ;-)

Reply:
Well, since the project has been going with no whitespace there, I'd ask to keep in line with that.

Reply:
fixed